### ISSUE TITLE
[Bug] (a11y) tabs tray: bad contrast for inactive mode icon in dark theme

### ISSUE THREAD

**Issue by cadeyrn** [mozilla-mobile/fenix#11561]
### Steps to reproduce
1. enable the dark theme
1. open the tabs tray

### Expected / Actual behavior
The mode icons (regular or private tabs) should have a sufficient color contrast ratio. While I think the icon of the active mode have a good contrast in the light and in the dark theme I can't say the same for the icon of the inactive mode. It's hard to see in the dark mode. In fact there is a big difference in the color contrast ratio when I compare the light and the dark theme:

Light theme: 2.85:1
Dark theme: 1.91:1

![Screenshot](https://user-images.githubusercontent.com/1100614/84588667-d2197f80-ae29-11ea-8a8b-1ac417ee4199.jpg)UNITO-UNDERSCORE!20200614-101951!
(In this screenshot the icon for private tabs is the inactive icon)

### Device information
* Android device: OnePlus 7T Pro McLaren Edition with Oxygen OS 10.0.9 (Android 10)
* Fenix version: master branch revision 7e6116512c354eb1d8524f8922f7970fe155c65c

**Comment by topotropic** [comment]
@AmyYLee here's a a11y contrast issue with the inactive PBM icon. Can you have a look? Thanks!

**Comment by AmyYLee** [comment]
@gabrielluong @topotropic - For deselected states can we keep it to Light Grey 5 (#FBFBFE) - This would also apply to the 3-dot menu as well. Thanks

**Comment by ebalazs-sv** [comment]
Verified as fixed on Nightly 200929 05:08 (Build #2015766697) GV 83.0a1 from 9/29 with Motorola Moto G6 (Android 8) and Samsung Galaxy Tab S3 (Android 8).
I will close this issue and remove the qa:needed label.

![rsz_screenshot_20200929-145100_firefox_nightly](https://user-images.githubusercontent.com/48799384/94555036-a64cc900-0263-11eb-970c-39f8a836ba3e.jpg)
